packages/server/src/api/routes/row.ts

```diff
@@ -269,6 +269,7 @@ router
 
 router.post(
   "/api/v2/views/:viewId/search",
+  paramResource("viewId"),
```

Comment:
Not sure this will actually work as expected. I will double check with my permissions changes once merged